client/index.js

```diff
@@ -1,14 +1,26 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
+import {applyMiddleware, compose, createStore} from 'redux'
 import {BrowserRouter as Router} from 'react-router-dom'
+import {Provider} from 'react-redux'
+import thunkMiddleware from 'redux-thunk'
 
 import App from './components/App'
+import reducer from './reducer'
```

Comment:
reducers please as it shall have more than one

Reply:
Made the changes